Makefile

```diff
@@ -30,9 +30,13 @@ mainnet_build:
 	export REACT_APP_EXPLORER_SERVICE_BASE_URL=https://explorer-service.hathor.network/; \
 	npm run build
 
-.PHONY: mainnet_s3_sync
+.PHONY: mainnet_s3_sync clear_cloudfront_cache
 mainnet_s3_sync:
 	aws s3 sync --delete ./build/ s3://hathor-mainnet-public-explorer
 
 .PHONY: mainnet_deploy
-mainnet_deploy: check_version check_tag mainnet_s3_sync
\ No newline at end of file
+mainnet_deploy: check_version check_tag mainnet_s3_sync
+
+.PHONY: clear_cloudfront_cache
+clear_cloudfront_cache:
+	aws cloudfront create-invalidation --distribution-id $$CLOUDFRONT_ID --paths "/index.html"
```

Comment:
Is paths "/index.html" enough? This wouldn't invalidate "/", right?

Reply:
We had made some tests in the past invalidating just `/index.html` and it worked.

I think `/` is not a path CloudFront is concerned for caching, since it's not a file. `/` is configured to serve the file `/index.html`, so invalidating `/index.html` is enough for `/`.

Reply:
Yes, as this is a Single Page App, update only the index.html is enough. JS and CSS files are generated with hash names that will be different.

Reply:
OK, I thought CloudFront didn't even know the filenames. So when accessing `/` it serves the `/index.html` file and then it looks on the cache. Fine for me.